src/raftstore/store/peer.rs

```diff
@@ -1144,6 +1151,10 @@ impl Peer {
             return Ok(RequestPolicy::ProposeNormal);
         }
 
+        if req.get_requests().is_empty() {
+            return Ok(RequestPolicy::Nop);
```

Comment:
How is it possible?

Reply:
An empty write or read. https://github.com/pingcap/tikv/blob/a08ed0db3f7ca8fecc143148c3da01ad15a943dc/tests/storage/test_raftkv.rs#L40